src/Components/Components/src/Routing/Router.cs

```diff
@@ -56,6 +63,16 @@ public class Router : IComponent, IHandleAfterRender, IDisposable
         /// </summary>
         [Parameter] public RenderFragment<RouteData> Found { get; set; }
 
+        /// <summary>
+        /// Get or sets the content to display when the upcoming route is loading.
+        /// </summary>
+        [Parameter] public RenderFragment Loading { get; set; }
```

Comment:
```suggestion
        /// <summary>
        /// Get or sets the content to display when asynchronous navigation is in progress.
        /// </summary>
        [Parameter] public RenderFragment Navigating { get; set; }
```

Reply:
I forgot to include this suggestion in the batch and I did make my modification in the local sources I'll be pushing.

I like the name `Loading` a little bit more because it's a lot more familiar to users. That might be a bad thing in this situation though because the term is overloaded. I'm fine with `Navigating` as a more descriptive term.

Reply:
The Navigating suggestion was because we don’t know what the developer is doing in the async callback. They might be loading something, or doing anything else.